README.md

```diff
@@ -1,15 +1,17 @@
 ## IO 500 in Four Easy Steps
 
+*NOTE: please use the "io500-app" repo for ISC20 and later list submissions*
```

Comment:
Is this note confusing?  They still use this one and it just clones the io500-app repo, no?  If I see this note, I think I might just stop reading and go straight to the other repo.

Reply:
Yeah, we could write something like the following instructions will clone also the io500-app

Reply:
then seems reading later that I am wrong, the io500-app repository should be used.

Reply:
Yes, the io500-app repo should be used, and the `prepare.sh` script there will download this in the background, like it does for IOR and pfind, so users shouldn't be downloading this directly anymore.  The note is there in case people come across this repo from older instructions, or if they have the io-500-dev repo checked out and update to get the latest version, so that they are aware of this change.